types/react-native-channel-plugin/index.d.ts

```diff
@@ -0,0 +1,577 @@
+// Type definitions for react-native-channel-plugin 0.7
+// Project: https://www.channel.io
```

Comment:
```suggestion
// Project: https://developers.channel.io/docs/react-native-quickstart
```
maybe that one is better suited for the types? (suggestion)

Reply:
this is better!